amd/src/stackjsvle.js

```diff
@@ -278,7 +278,7 @@ define([
         // the core libraries, here is one implementation that shows what we
         // are looking for.
 
-        // TODO: look into replacing this with DOMPurify or some such.
+        // TO-DO: look into replacing this with DOMPurify or some such.
```

Comment:
This shouldn't really have been touched but maybe best to leave these TODO changes for consistency (and also in the other affected JS files.)